src/main/java/com/atlassian/db/replica/internal/SqlQuery.java

```diff
@@ -38,9 +39,17 @@ public boolean isInsert() {
     }
 
     public boolean isSqlSet() {
+        return compatibleWithPreviousVersion ? isSqlSet_old() : isSqlSet_new();
+    }
+
+    private boolean isSqlSet_old() {
         return sql.startsWith("set");
     }
 
+    private boolean isSqlSet_new() {
+        return sql.startsWith("set") || sql.startsWith("SET");
```

Comment:
how about `sEt` or `SeT` :lol

Reply:
We had the discussion before 😉. Db-replica, in the first place, aims for low overhead, and correctness is in second place. It will not work correctly for `SeT`, but it's not worth reducing the performance of every query to cover this corner case.

Reply:
> We had the discussion before
Yeah, we had, bad luck that we didn't find SET that time.